src/TinyTrainable.h

```diff
@@ -26,12 +26,14 @@ class TinyTrainable {
       }
     };
 
+  // TODO: why protected instead of private
   protected:
+    // TODO: why static void instead of void
```

Comment:
i did this because you only need to run it once per arduino even if you have multiple instances of the instrument classes (it doesn't need to be an instance method). but i realize that i wasn't consistent and other setup functions that don't interact with instance variables aren't static. i can make those static too or just make this an instance method?

Reply:
okie i deleted this line and added your comment to the docs/software.md

Reply:
i just want to have consistency, and also to understand each line, i didn't know this difference and i want to include those on the docs :)